### PR TITLE
Update menu prices

### PR DESCRIPTION
**Pull request by SebastianSalasDCI**
I changed the prices of the food in the menu file.

### CODE REVIEW

File: menu.html [hunk 1]
```diff
@@ -106,7 +106,7 @@ <h1>Kyoto Dining</h1>
                                 sashimi (6 slices)
                             </td>
                             <td>
-                                Euro 450
+                                Euro 650
```
Comment (SuperMario4ever) [inline menu.html:109]: I think that is too expensive for sashimi!

File: menu.html [hunk 2]
```diff
@@ -117,7 +117,7 @@ <h1>Kyoto Dining</h1>
                                 nigiri tuna and salmon
                             </td>
                             <td>
-                                Euro 150
+                                Euro 350
```
Comment (SuperMario4ever) [inline menu.html:120]: No way!

File: menu.html [hunk 3]
```diff
@@ -128,7 +128,7 @@ <h1>Kyoto Dining</h1>
                                 miso soup
                             </td>
                             <td>
-                                Euro 35
+                                Euro 25
```
Comment (SuperMario4ever) [inline menu.html:131]: What a bargain! Good idea, Sebastian.

File: menu.html (hunk 1 above)
Comment (SebastianSalasDCI) [reply menu.html:109]: No, it is actually a good price

Comment (SuperMario4ever) [reply menu.html:109]: No one might buy them anymore.

**Pull request merged**